task/kaniko/0.1/kaniko.yaml

```diff
@@ -59,7 +59,7 @@ spec:
       runAsUser: 0
   - name: write-digest
     workingDir: $(workspaces.source.path)
-    image: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/imagedigestexporter:v0.15.2
```

Comment:
Hum, I guess we *may* want to fix this by adding a `securityContext` `runAsUser: 0` instead, but could be a follow-up.

Reply:
or probably I can fix that in this PR itself :grin:  because version downgrading is not the better solution I guess :thinking: